test/test_optimizers.py

```diff
@@ -390,12 +390,12 @@ def test_sbr_accurate():
     [
         dict(thresholder="l1", threshold=0),
         dict(thresholder="l1", threshold=1e-5),
-        dict(thresholder="weighted_l1", thresholds=np.zeros((9, 3))),
-        dict(thresholder="weighted_l1", thresholds=1e-5 * np.ones((9, 3))),
+        dict(thresholder="weighted_l1", thresholds=np.zeros((3, 9))),
+        dict(thresholder="weighted_l1", thresholds=1e-5 * np.ones((3, 9))),
         dict(thresholder="l2", threshold=0),
         dict(thresholder="l2", threshold=1e-5),
-        dict(thresholder="weighted_l2", thresholds=np.zeros((9, 3))),
-        dict(thresholder="weighted_l2", thresholds=1e-5 * np.ones((9, 3))),
+        dict(thresholder="weighted_l2", thresholds=np.zeros((3, 9))),
+        dict(thresholder="weighted_l2", thresholds=1e-5 * np.ones((3, 9))),
```

Comment:
I thought you added a transpose in SR3?

Reply:
I undid the changes in the previous commit. If you compare to the main branch, you can see that the original shape is (3,9)